docs/source/stepfunctions.rst

```diff
@@ -0,0 +1,118 @@
+.. _stepfunctions:
+
+CfnCluster Stepfunctions
+########################
+
+Why Stepfunctions
+=================
+
+* Allows for complex workflows with CfnCluster
+* Handles cluster creation, teardown, and updates
+* Useful for conditional automated job execution
+* Has better interfacing with other AWS services
+
+.. image:: https://s3.amazonaws.com/global-cfncluster/doc-images/parallel_job.gif
+
+Using CfnCluster Step Functions
+===============================
+
+.. image:: https://s3.amazonaws.com/global-cfncluster/doc-images/command_start.gif
+
+1. Configure CfnCluster configuration file with :code:`cfncluster configure` or manually
```

Comment:
you can use \`\` for inline code

Reply:
are you telling my I could have saved 2 minutes while writing these docs??

Reply:
that's four extra characters per inline code

Reply:
sigh

Reply:
more than 4, you don't need `:code:`